src/utils.ts

```diff
@@ -407,23 +407,27 @@ export const tagBackportReviewers = async ({
   user?: string;
 }) => {
   const reviewers = [];
+  const teamReviewers = [];
 
   if (DEFAULT_BACKPORT_REVIEW_TEAM) {
-    reviewers.push(DEFAULT_BACKPORT_REVIEW_TEAM);
+    // Optionally request a default review team for backports.
+    // Use team slug value. i.e electron/wg-releases => wg-releases
+    const slug = DEFAULT_BACKPORT_REVIEW_TEAM.split('/')[1];
+    teamReviewers.push(slug ? slug : DEFAULT_BACKPORT_REVIEW_TEAM);
```

Comment:
```suggestion
    const slug = DEFAULT_BACKPORT_REVIEW_TEAM.split('/')[1] || DEFAULT_BACKPORT_REVIEW_TEAM;
    teamReviewers.push(slug);
```

Suggestion for readability. Functionally equivalent, but I think it's more readable than a ternary in this case.

Reply:
Agreed! Change made